pkgs/tools/system/hostctl/default.nix

```diff
@@ -0,0 +1,41 @@
+{ buildGoModule, fetchFromGitHub, lib, installShellFiles }:
+
+buildGoModule rec {
+  pname = "hostctl";
+  version = "1.0.14";
+
+  src = fetchFromGitHub {
+    owner = "guumaster";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "02bjii97l4fy43v2rb93m9b0ad8y6mjvbvp4sz6a5n0w9dm1z1q9";
+  };
+
+  vendorSha256 = "1lqk3cda0frqp2vwkqa4b3xkdw814wgkbr7g9r2mwxn85fpdcq5c";
+
+  doCheck = false;
```

Comment:
Why? If there is a good reason to disable the tests please explain it in a comment.

Reply:
I have not even reasoned about it. Just copied from another place :wink:  &mdash; legacy bad practice, I guess.